### ISSUE TITLE
discogs.com script: Failure to import all tracks

### ISSUE THREAD

**Issue by orrigogp** [murdos/musicbrainz-userscripts#348]
When a release in discogs.com has tracks grouped under one name, only the group is imported as track in musicbrainz, with the duration for that track equal to the total duration of the tracks in that group.
Example: https://www.discogs.com/Carissimi-I-Madrigalisti-Ambrosiani-Gianluca-Capuano-Missa-Lhomme-Arm%C3%A9-Oratorio-Judicium-Extremum-M/release/7490445

![Screenshot 2020-12-21 204355](https://user-images.githubusercontent.com/76186338/102838960-6feb9980-43cd-11eb-90fa-8774ff947a7a.png)
![Screenshot 2020-12-21 204446](https://user-images.githubusercontent.com/76186338/102838983-7a0d9800-43cd-11eb-8ee0-4fe8cecffa35.png)

**Comment by orrigogp** [comment]
After a bit of looking at the script code, it seems the problem lies in the way subtracks are imported. The way this is done it is not compatible with the way classical releases are normally treated. Is it possible to have a version of the script that import subtracks as tracks, with the main track name appended to each subtrack?

**Comment by kaysond** [comment]
Bumping. Running into this issue myself

**Comment by jesus2099** [comment]
Isn't this ticket a **duplicate of #138**?

**Comment by orrigogp** [comment]
Yes! Thank you for pointing that out! At the time I couldn’t find it.
